### PR TITLE
@albeanth cleaning and adding Doug's original website wiki work for 1…

### PR DESCRIPTION
**Pull request by albeanth**
…0.5.2016 RTRP meeting

I have also included a new ./references/ directory where we can stash .pdf's of various references. There is currently a/the beamer user guide and the pro git github book. 

Check it out, let me know what you think.

### CODE REVIEW

File: wiki/index.md [hunk 1]
```diff
@@ -1,39 +1,65 @@
...
-* [LaTeX Demo](../users/woodsdou/LatexDemo.zip)
+  - A new(er) piece of software that definitely seems interesting!
 
-## Other
+## Version Control
 * [GitHub](http://www.github.com)
+  - Your one-stop-shop for version control. Students can obtain a paid, premium account for free through [GitHub's education](https://education.github.com/) program.  
+  - The learning curve for using Git effectively can be a little daunting. Thankfully, GitHub has [free training](https://services.github.com/training/). Myself (Tony) and a few others took the GitHub for Developer's training and it is a *game changer*.
+
+## Editors
```
Comment (dougwoods3) [inline wiki/index.md:56]: good idea to add editors.

File: wiki/index.md [hunk 2]
```diff
@@ -1,39 +1,65 @@
...
 ---
 More coming soon...
 ================================
 
 # Available Software
 Some software available through [Citrix](http://it.engineering.oregonstate.edu/citrix).
 
+## Code Reference Resources
+* [C++](http://www.learncpp.com/)
+* [Python3](https://docs.python.org/3/), [Python2](https://docs.python.org/3/)
```
Comment (dougwoods3) [inline wiki/index.md]: these links are the same.

**Comment by dougwoods3** [comment]
I debated adding some visualization software to the Wiki. I use VisIt but it's hard to use without MFEM so I didn't add it.

File: wiki/index.md (hunk 2 above)
Comment (albeanth) [reply wiki/index.md]: good catch. I'll fix that.

**Comment by albeanth** [comment]
@dougwoods3 pull these updates and add the visualization stuff for visit. Once you get it going for Visit I can add some brief things for ParaView. If @harterj wants to expand upon it (since he has more experience with the ins and outs than I do) he totally could.

**Pull request merged**